bin/src/property.dart

```diff
@@ -0,0 +1,110 @@
+library polymerjs.property;
+
+import 'matchers.dart';
+import 'common.dart';
+
+class Property {
+  String name;
+  String dartName;
+  List<String> doc;
+  String value;
+  String observer;
+  String type = 'String';
+  bool notify = false;
+  bool reflectToAttribute = false;
+  bool readOnly = false;
+  List<String> output = [];
+
+  Property(this.name, String rawdocs, String rawinfo) {
+    // TODO what to do with private variables ?
```

Comment:
If they're private in the polymer definition but we don't want to have them as private for dart? You could just emit the dart version with an annotation (`@private`) or some dartdoc saying it was private in the definition.